### PR TITLE
Updated Registration details

### PR DESCRIPTION
**Pull request by karegapauline**

### CODE REVIEW

File: pages/en/register.markdown [hunk 1]
```diff
@@ -5,13 +5,41 @@ permalink: /register/
...
+Papers will be selected via a rigorous double-blind peer-review process on OpenReview, with submissions and reviews being private until a decision is made. Reviews and discussions of the accepted papers will be made available after acceptance. All accepted papers will be presented at the Conference as contributed talks.
 
-<!--Add a call for abstract submissions-->
- Registration opens on 15 March 2022.
+### Key dates
+The planned dates are as follows:
+- Online conference sessions: 26-29 April 2022
+- Abstract submission deadline: 4th April 2022
+- Abstract notification: 18th April 2022
+- Registration open: 10th March 2022
```
Comment (bkmgit) [inline pages/en/register.markdown:39]:
```suggestion
- Registration open: 16th March 2022
```

File: pages/en/register.markdown [hunk 2]
```diff
@@ -5,13 +5,41 @@ permalink: /register/
...
+
+### Submission Link
+Submissions are done in Open Review. Follow this link: [Register Here](https://openreview.net/group?id=bioinformatics-hub-ke.github.io/BOSS/2022/Conference)
```
Comment (bkmgit) [inline pages/en/register.markdown:42]:
```suggestion
Submissions are done in Open Review, [submit here](https://openreview.net/group?id=bioinformatics-hub-ke.github.io/BOSS/2022/Conference).
```

File: pages/en/register.markdown [hunk 3]
```diff
@@ -5,13 +5,41 @@ permalink: /register/
...
+
+### Submission Guidelines
+Visit our wiki pages for Abstract submission guidelines
```
[the remark this answers is not in the document]
Comment (karegapauline) [reply pages/en/register.markdown:45]: They are quite a lot of details in the guide

[the remark this answers is not in the document]
Comment (bkmgit) [reply pages/en/register.markdown:45]: Then perhaps add a link to the wiki? Expect that viewing on the webpage is nice than the wiki.

[the remark this answers is not in the document]
Comment (kipkurui) [reply pages/en/register.markdown:45]: I think we can just add them at the end of the Call for Abstracts

[the remark this answers is not in the document]
Comment (bkmgit) [reply pages/en/register.markdown:45]: However, this should not be a point of contention.

[the remark this answers is not in the document]
Comment (bkmgit) [reply pages/en/register.markdown:45]: Let me know when ready for another review.

**Review by kipkurui — CHANGES_REQUESTED** [review]
@karegapauline, let's add the guide at the end

File: pages/en/register.markdown (hunk 3 above)
[the remark this answers is not in the document]
Comment (kipkurui) [reply pages/en/register.markdown:45]: I agree, we can merge as is then can update later.

**Review by kipkurui — APPROVED** [review]

**Pull request merged**